ci-operator/step-registry/single-node/e2e/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
I created the owners alias of assisted-cicd to reduce these duplicate files

Reply:
This test is for single-node and doesn't deal with assisted, so it's owned by different people. But I wasn't aware of OWNER_ALIASES, thanks, I'll create one for the single node team, we do have a lot of duplication in single-node related steps. I'll fix that in a separate PR.